packages/guideline-blocks-settings/src/components/RichTextEditor/plugins/LinkPlugin/FloatingLink/EditLinkModal/EditModal.tsx

```diff
@@ -21,7 +21,7 @@ export const EditModal = ({ editButtonProps, unlinkButtonProps }: EditModalProps
 
     return (
         <FloatingModalWrapper data-test-id="floating-link-edit" padding="16px" minWidth="400px">
-            <span data-test-id="preview-link-flyout" className="tw-flex tw-justify-between tw-items-center">
+            <span data-test-id="preview-link-flyout" className="tw-flex tw-justify-between tw-items-center tw-gap-2">
                 <span className="tw-pointer-events-none">{url}</span>
```

Comment:
The preview here should be linked: https://frontify.slack.com/archives/CCXT7AFD0/p1720776076256159

Reply:
Nice. Created a ticket for that, let me tackle that in a different pr
https://app.clickup.com/t/86953m74n